src/components/blocked-patron/BlockedPatron.test.ts

```diff
@@ -16,7 +16,7 @@ describe("Patron page", () => {
     cy.visit("/iframe.html?path=/story/apps-loan-list--loan-list-entry");
     cy.get(".modal").should("not.exist");
   });
-
+  // TODO: Explain E
```

Comment:
I am sorry this is not good enough, I think. The best solution would be to have a global object/enum AND type with these values.
But second/third-best is to write a TODO. But then at least you should still specify what E,W,A etc are... To help future developers.

Reply:
donnnneeee